WIP(web-diary): react app basic structure,style,configs

gochicus opened the pull request:
affects: web-diary

ISSUES CLOSED: #56

gochicus committed:
WIP(web-diary): react app basic structure,style,configs

affects: web-diary

ISSUES CLOSED: #56

gochicus committed:
chore: configs changed

ISSUES CLOSED: #56

gochicus committed:
feat(web-diary): import and export added

affects: @practice/web-diary

ISSUES CLOSED: #56

gochicus committed:
chore: eslint config changed

ISSUES CLOSED: #56

ad21ewda1231dsa123das12 commented on practice/web-diary/.gitignore:
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```
.gitignore уже есть в корне, в нём нет необходимости

ad21ewda1231dsa123das12 commented on practice/web-diary/README.md:
```diff
@@ -0,0 +1,70 @@
+
+### `yarn build` fails to minify
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
```
убери дефолтное описание cra, и сделай подходящее твоему проекту

ad21ewda1231dsa123das12 commented on practice/web-diary/package.json:
```diff
@@ -0,0 +1,34 @@
+    "react-dom": "^17.0.1",
+    "react-scripts": "4.0.3",
+    "typescript": "^4.2.2"
+
```
строка пропущена

ad21ewda1231dsa123das12 commented on practice/web-diary/public/index.html:
```diff
@@ -0,0 +1,12 @@
+<body>
+<div id="root"></div>
+</body>
+</html>
```
no newline at EOF

ad21ewda1231dsa123das12 commented on practice/web-diary/src/style/app-header.css:
```diff
@@ -0,0 +1,16 @@
+.app-header {
+  align-items: flex-end;
+  justify-content: space-between;
+
```
https://github.com/Atlantis-Academy/frontend-study-ikorol/pull/60#discussion_r584090607

gochicus replied:
@themetronome   не понял что ты имеешь в виду, в коменте говорится про имена компонентов, а здесь стили.

ad21ewda1231dsa123das12 replied:
@gochicus в коменте говорится про пустую строку, идентичная ситуация

gochicus replied:
@themetronome извини, действительно не доглядел

ad21ewda1231dsa123das12 commented on practice/web-diary/src/style/app-header.css:
```diff
@@ -0,0 +1,16 @@
+}
+
+.app-header h1 {
+  /* width: 150px;
+  height: 150px; */
```
https://github.com/Atlantis-Academy/frontend-study-ikorol/pull/60#discussion_r584090851

ad21ewda1231dsa123das12 commented on practice/web-diary/src/style/post-list-item.css:
```diff
@@ -0,0 +1,54 @@
+  border: none;
+  cursor: pointer;
+}
+.app-list-item button:focus {
+    box-shadow: none;
+    outline: none;
+}
+.app-list-item .btn-star {
+    color: #FFD700;
+}
+.app-list-item .btn-trash {
+    color: red;
+}
+.app-list-item .fa-heart {
+    width: 35px;
+    height: 35px;
+    text-align: center;
+    line-height: 35px;
+    font-size: 16px;
+    color: red;
+    transition: 0.3s all;
+    transform: translateX(30px);
+    opacity: 0;
+}
```
https://github.com/Atlantis-Academy/frontend-study-ikorol/pull/60#discussion_r584090382

ad21ewda1231dsa123das12 commented on practice/web-diary/src/style/post-list.css:
```diff
@@ -0,0 +1,7 @@
+.app-list {
+    margin-top: 50px;
+}
+.app-list .list-group-item {
```
https://github.com/Atlantis-Academy/frontend-study-ikorol/pull/60#discussion_r584089775

ad21ewda1231dsa123das12 reviewed (changes requested):
В **post-status-filter** пустой файл со стилями, если он нигде не используется, полагаю он не нужен

gochicus committed:
style(web-diary): whitespaces added, empty lines deleted

affects: @practice/web-diary

ISSUES CLOSED: #56

gochicus committed:
style(web-diary): empty line deleted

affects: @practice/web-diary

ISSUES CLOSED: #56

gochicus committed:
refactor(web-diary): import deleted, Readme.md rewritten

affects: @practice/web-diary

ISSUES CLOSED: #56

gochicus committed:
refactor(web-diary): react-app structure refactored

affects: @practice/web-diary, @components/app-header, @components/app, @components/post-add-form,
@components/post-list-item, @components/post-item, @components/filter, @components/search,
@ui/button, @ui/input

ISSUES CLOSED: #56

gochicus committed:
refactor(web-diary): code structure changed, packages import changed

affects: @practice/web-diary, @app-launcher/app, @ui/app-header, @ui/button, @ui/input,
@ui/post-add-form, @ui/post-list-item, @ui/post-list, @ui/filter, @ui/search

ISSUES CLOSED: #56

gochicus committed:
style(web-diary): whitespaces added in package.json files

affects: @practice/web-diary

ISSUES CLOSED: #56

gochicus committed:
chore(web-diary): dependency added in package.json

affects: @practice/web-diary

ISSUES CLOSED: #56
